afids_utils/tests/test_regRF.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import numpy as np
+import polars as pl
+from hypothesis import given
+from hypothesis import strategies as st
+from hypothesis.extra.numpy import arrays
+from numpy.typing import NDArray
+
+from afids_utils.features import regRF
+from afids_utils.tests.strategies import nii_img
+
+
+class TestIntegralVolume:
+    @given(nii_img=nii_img())
+    def test_integral_float_volume(self, nii_img: NDArray[np.float_]):
+        # Compute expected result
```

Comment:
I think this kind of test where the function being tested is reimplemented in exactly the same way within the test should be avoided. What we could do is check for every voxel (or some random subset of voxels) (i, j, k) in `iv_img` that it's value is equal to (something like) `nii_img[:i, :j, :k].sum()`, which I think is slightly different. (We should keep the type check either way)